### ISSUE TITLE
[V6] Set value dynamically in Field Props

### ISSUE THREAD

**Issue by felisio** [erikras/redux-form#1274]
I'm taking a picture coming from the webcam, an i need set in field, how to dynamically insert in the field? is my code now:

```js
printAvatar(){
    if(this.state.photo.preview || this.props.photoValue){
      return(
        <Avatar
          src={this.state.photo.preview}
          size={100}
          style={styles.avatar}
          onTouchTap={this.pictureOpen}
        />
      );
    }else{
      return(
        <IconButton
          iconStyle={styles.largeIcon}
          style={styles.largeButton}
          onTouchTap={this.pictureOpen}
        >
          <ActionAccountCircle />
        </IconButton>
      );
    }
  }

```

**Comment by erikras** [comment]
I'm not sure I understand, as I don't see any form or fields in your code. What kind of field is it? A file?

To set the value, you can dispatch a [`change(form, field, value)`](http://redux-form.com/6.0.0-rc.1/docs/api/ActionCreators.md/#-change-form-string-field-string-value-any-) action.

**Comment by felisio** [comment]
@erikras That's what I was looking for, thanks. I still can not use the [InitialValues](http://redux-form.com/6.0.0-rc.1/examples/initializeFromState/) and [sync validate](http://redux-form.com/6.0.0-rc.1/examples/syncValidation/), when it will be fixed? in the v5 works fine...

```js
const validate = formProps => {
  const errors = {}
  const requiredFields = [ 'name', 'email', 'consultant' ]
  requiredFields.forEach(field => {
    if (!formProps[ field ]) {
      errors[ field ] = 'Required'
    }
  })
  if (formProps.email && !/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(formProps.email)) {
    errors.email = 'Invalid email address'
  }
  return errors
}

FormProspect = reduxForm({
  form: 'cardprospect',
  validate
})(FormProspect);
const selector = formValueSelector('cardprospect');

const mapStateToProps = (state) => ({
  photoValue: selector(state, 'photo'),
  result: state.prospects.result,
  initialValues: state.prospects.prospect,
  loading: state.prospects.loading,
  consultantList: state.employees.consultantList
});

FormProspect = connect(mapStateToProps, actions)(FormProspect)

export default FormProspect;

```

**Comment by erikras** [comment]
Nothing really stands out to me as wrong about this code. Make sure you use the Redux Dev Tools and examine exactly what is going on in your redux store. For example, does `state.propsects.prospect` really exist?

**Comment by felisio** [comment]
Thanks, the ``` state.propsects.prospect ```  It is called and return correct data, I will verify the data again

**Comment by erikras** [comment]
I have confirmed that there is a sync validation bug in v6rc2. So you are not insane. 😄

**Comment by erikras** [comment]
Should be fixed in [`6.0.0-rc.3`](https://github.com/erikras/redux-form/releases/tag/v6.0.0-rc.3).
